scrapper.py

```diff
@@ -1,7 +1,15 @@
 """
 Crawler implementation
 """
-
+import json
```

Comment:
1. imports in alphabetical order
2. imports in following order: Python standard library, empty line, external libraries, empty line, local modules